gradle-plugin/it/src/test/java/org/eclipse/jkube/gradle/plugin/tests/JavaOptionsEnvMergeIT.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.gradle.plugin.tests;
+
+import net.minidev.json.parser.ParseException;
+import org.eclipse.jkube.kit.common.ResourceVerify;
+import org.gradle.testkit.runner.BuildResult;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class JavaOptionsEnvMergeIT {
+  @Rule
+  public final ITGradleRunner gradleRunner = new ITGradleRunner();
+
+  @Test
+  public void k8sResource_whenRun_thenGeneratedManifestShouldHaveMergedEnvVariables() throws IOException, ParseException {
+    // When
+    final BuildResult result = gradleRunner.withITProject("java-options-env-merge").withArguments("k8sResource").build();
```

Comment:
ocResource should be invoked and openshift.yml checked too

Reply:
I found this test only in kubernetes-maven-plugin module. Hence I only ported kubernetes only

Reply:
Issue mentions both.
The maven test is more expensive, since our current project structure separates these tests in two modules.
I understand that performing the OpenShift validation should be quite cheap in the Gradle version. If I'm not mistaken, it should only involve calling the additional task which should execute really quick (ms) (the Gradle daemon is already started for the test).